src/Drivers/GuzzleDriver.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ragnarok\Bifrost\Drivers;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Psr7\Request as Psr7Request;
+use Psr\Http\Message\ResponseInterface;
+use Ragnarok\Bifrost\DriverInterface;
+use Ragnarok\Bifrost\Request;
+use React\Promise\ExtendedPromiseInterface;
+use React\Promise\Promise;
+
+class GuzzleDriver implements DriverInterface
+{
+    public function __construct(private Client $client = new Client())
```

Comment:
```suggestion
    public function __construct(
        private Client $client = new Client()
    )
```

Different coding style compared to the [React Driver's constructor](https://github.com/dc-Ragnarok/Bifrost/pull/9/commits/3ef66d518808ceb8f2faf6125b9607837c4a22ad#diff-0b3939665ea1190ccaae339c078f88d367fe2d2e523724ddf47bad9974be8e63R8)